package.json

```diff
@@ -37,7 +37,8 @@
     "react-dom": "^18.2.0",
     "react-query": "^3.39.1",
     "react-router-dom": "^6.3.0",
-    "yup": "^0.32.11"
+    "yup": "^0.32.11",
+    "serverless-single-page-app-plugin": "file:./serverless-single-page-app-plugin"
```

Comment:
Not sure this line makes sense. That was in the video, but the repo's structure is quite different. Let's clarify on Q&A session.

Reply:
you are right, that line is not necessary, I removed it and the publish to aws still works fine.